e2e-tests/ukef/cypress/e2e/journeys/utilisation-reports/utilisation-report-submission.spec.js

```diff
@@ -0,0 +1,235 @@
+      expectedParsedValuesForReportRow.reportedFees,
+    );
+    cy.assertText(
+      tfmPages.utilisationReportPage.premiumPaymentsTab.premiumPaymentsTable.reportedPaymentsByRowIndex(premiumPaymentsRowIndex),
+      expectedParsedValuesForReportRow.reportedPayments,
+    );
+    cy.assertText(
+      tfmPages.utilisationReportPage.premiumPaymentsTab.premiumPaymentsTable.totalReportedPaymentsByRowIndex(premiumPaymentsRowIndex),
+      expectedParsedValuesForReportRow.reportedPayments,
+    );
+  };
+
+  /**
+   * Asserts the values in a given row of the utilisation table match the expected
+   * values from the report as defined in february2023ExpectedValues
+   * @param {number} utilisationTableRowIndex - The index of the row of utilisation table
+   * @param {object} expectedParsedValuesForReportRow - The expected values to be found in the row
+   */
+  const assertUtilisationTableRowContainsExpectedValues = (utilisationTableRowIndex, expectedParsedValuesForReportRow) => {
+    tfmPages.utilisationReportPage.utilisationTab.table.rowByIndex(utilisationTableRowIndex).within(() => {
+      cy.assertText(tfmPages.utilisationReportPage.utilisationTab.table.facilityId(), expectedParsedValuesForReportRow.facilityId);
+      cy.assertText(tfmPages.utilisationReportPage.utilisationTab.table.exporter(), expectedParsedValuesForReportRow.exporter);
+      cy.assertText(tfmPages.utilisationReportPage.utilisationTab.table.feesPayable(), expectedParsedValuesForReportRow.reportedFees);
+      cy.assertText(tfmPages.utilisationReportPage.utilisationTab.table.feesAccrued(), expectedParsedValuesForReportRow.feesAccrued);
+      cy.assertText(tfmPages.utilisationReportPage.utilisationTab.table.utilisation(), expectedParsedValuesForReportRow.utilisation);
+    });
+  };
+
+  const testFebruary2023ReportUploadWithFileType = (filetype) => {
+    const filename = filetype === 'csv' ? 'valid-utilisation-report-February_2023_monthly.csv' : 'valid-utilisation-report-February_2023_monthly.xlsx';
+
+    //---------------------------------------------------------------------------------
+    // Portal payment report officer submits utilisation report
+    //---------------------------------------------------------------------------------
+    cy.login(BANK1_PAYMENT_REPORT_OFFICER1);
+    cy.visit(relative('/utilisation-report-upload'));
+
+    portalPages.utilisationReportUpload.utilisationReportFileInput().attachFile(filename);
+    cy.clickContinueButton();
+    portalPages.confirmAndSend.confirmAndSendButton().click();
+
+    //---------------------------------------------------------------------------------
+    // PDC_RECONCILE user can login to TFM and view the data from the submitted report
+    //---------------------------------------------------------------------------------
+    cy.clearCookie('dtfs-session');
+    cy.clearCookie('_csrf');
+    cy.getCookies().should('be.empty');
+
+    cy.forceVisit(TFM_URL);
```

Comment:
Why do we need `forceVisit`?

Reply:
I'm not sure, I copied it from the other tests in the 'ukef' set of tests since they all do it so I just assumed it was necessary. I'll have a see what happens without it.